google-cloud-firestore/src/main/java/com/google/cloud/firestore/CustomClassMapper.java

```diff
@@ -305,7 +305,7 @@ static Object convertToPlainJavaTypes(Object object) {
       try {
         result =
             (rawType == Map.class)
-                ? new HashMap<String, Object>()
+                ? new HashMap<>()
```

Comment:
I know it's not needed, but do you think we could gain any readability here from being explicit?

Reply:
nvm, the code above was hidden. This is fine 👍